### ISSUE TITLE
Styles for E-Mails leaks into admin UI

### ISSUE THREAD

**Issue by HoffmannP** [moodle-an-hochschulen/moodle-theme_boost_union#413]
Styles for e-mails using the language settings `templateemailhtmlprefix` leak into the complete admin page at `Appearance > Themes > Boost Union > Look`.

Add e.g.
```
<style>*{text-decoration: line-through 3px red wavy;}</style>
```
into your mails language string and enjoy.

**Comment by whuml** [comment]
Hi @HoffmannP 
we reproduced your finding and enjoyed the red wavy lines in boost unions admin sites. Thanks for reporting the bug :-)
Moodle 4.2.3+ (Build: 20231020)  / theme_boost_union | v4.2-r3 (2023090102)

**Comment by abias** [comment]
This was fixed in #466
